lib/ddtrace/contrib/elasticsearch/core.rb

```diff
@@ -0,0 +1,61 @@
+require 'uri'
+require 'ddtrace/pin'
+require 'ddtrace/ext/app_types'
+
+URL = 'elasticsearch.url'.freeze
+METHOD = 'elasticsearch.method'.freeze
+TOOK = 'elasticsearch.took'.freeze
+PARAMS = 'elasticsearch.params'.freeze
+BODY = 'elasticsearch.body'.freeze
+
+DEFAULTSERVICE = 'elasticsearch'.freeze
+
+module Datadog
+  module Contrib
+    module Elasticsearch
+      # Elastic Search integration.
+      module TracedClient
+        def initialize(*args)
```

Comment:
Do you want to pass **kwargs through too? Even if now used now, it might make it more future proof.